src/libs/_tests/runtime-utils.test.js

```diff
@@ -272,7 +272,7 @@ const cromwellProvisioningDiskUpdatedPd = {
   zone: 'us-central1-a'
 }
 
-const jupyterStopped = {
+const jupyter1 = {
```

Comment:
If the fact that it is running is important, I would make that part of the variable name (`jupyterRunning`).

Reply:
Or consider always overriding the status where it is used for clarity (I see below that you override this for Stopped).